src/engine.rs

```diff
@@ -698,10 +717,24 @@ mod tests {
         sub_dir.push("raft-engine");
         let cfg = Config {
             dir: sub_dir.to_str().unwrap().to_owned(),
+            enable_log_recycle: false,
```

Comment:
One point, if `recycling` enabled here, `ObfuscatedFs` would cause the time cost of preparing stale files tool high. 

And I also add an extra `cfg_recycle` for testing starting Engine with `enable-log-recycle == true` here.